lib/vsc/install/ci.py

```diff
@@ -150,12 +143,11 @@ def gen_tox_ini():
     # list of Python environments in which tests should be run
     envs = []
 
-    # don't run tests with Python 2 if 'py3_only' is set in vsc-ci.ini
+    # give error if 'py3_only' is set to false in vsc-ci.ini
     if not vsc_ci_cfg[PY3_ONLY]:
-        envs.append('py27')
+        logging.error("package is not py3_only. Not supported. Please set 'py3_only'. will ignore python2.")
```

Comment:
why should we even set it? it is the new default. `ci` module should even remove it, no?